client/reader/following/main.jsx

```diff
@@ -7,6 +9,7 @@ function FollowingStream( { ...props } ) {
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
 		<Stream { ...props }>
+			{ config.isEnabled( 'reader/editor' ) && <ReaderPostEditor /> }
```

Comment:
Is this something we should async load? 

Otherwise, we'll bundle this for everyone, including production, even though we don't really render it.

Reply:
Great point, now updated to async load.